### PR TITLE
New language ca (catalan)

### PR DESCRIPTION
**Pull request by mrestivill**
Added new language file obtained from other repo where is already translated: https://github.com/mcguirepr89/BirdNET-Pi?tab=readme-ov-file#internationalization zipped file in https://github.com/mcguirepr89/BirdNET-Pi/tree/main/model

I don't know if some configuration is missing to be able to activate this new language.

Thanks

### CODE REVIEW

**Comment by woheller69** [comment]
it has a wrong number of lines...
Must be 6522

**Comment by mrestivill** [comment]
updated file with 6522 lines

File: labels/V2.4/BirdNET_GLOBAL_6K_V2.4_Labels_ca.txt [hunk 1]
```diff
@@ -1,6362 +1,6522 @@
...
-Calandrella cinerea_Terrerola de casquet
+Calandrella brachydactyla_Terrerola Comuna
+Calandrella cinerea_Terrerola de Capell Rogenc
+Calandrella dukhunensis_Terrerola Oriental
 Calcarius lapponicus_Repicatalons de Lapònia
-Calcarius ornatus_Repicatalons pitnegre
+Calcarius ornatus_Repicatalons Pitnegre
 Calcarius pictus_Repicatalons de Smith
-Calendulauda africanoides_Alosa lleonada
+Calendulauda africanoides_notfound
```
Comment (Josef-Haupt) [inline labels/V2.4/BirdNET_GLOBAL_6K_V2.4_Labels_ca.txt]: If there is no translation, please use the english name from the english label file

Comment (woheller69) [reply labels/V2.4/BirdNET_GLOBAL_6K_V2.4_Labels_ca.txt]: @mrestivill updated these already in the latest commit

**Comment by Josef-Haupt** [comment]
I added your translation to the new PR. A lot has changed in the structure of this project, so I'm closing this PR

**Pull request closed without merging**